rbc/typesystem.py

```diff
@@ -475,12 +544,16 @@ def is_string(self):
 
     @property
     def is_bool(self):
-        return self.is_atomic and self[0] == 'bool'
+        return self.is_atomic and self[0].startswith('bool')
 
     @property
     def is_char(self):
         return self.is_atomic and self[0].startswith('char')
 
+    @property
+    def is_aggregate(self):
+        return self.is_struct or self.is_function
+
```

Comment:
A function is an aggregate type?

Reply:
no: https://llvm.org/docs/LangRef.html#aggregate-types
will fix it..

Reply:
Fixed.